spec/export/index.md

```diff
@@ -267,27 +268,30 @@ Content-Type: application/json
 - HTTP status code of ```5XX```
 - ```Content-Type``` header of ```application/json```
 - The server SHALL return a FHIR OperationOutcome resource in JSON format
-- The choice of when to determine that an export job has failed in its entirety (error status) vs returning a partial success (complete status) is left up to the implementer.
 
 	Example OperationOutcome response body:
 
   ```
+  Status: 500 Internal Server Error
+  Content-Type: application/json
+  Retry-After: 120
+
   {
     "resourceType": "OperationOutcome",
-    "id": "101",
+    "id": "1",
     "issue": [
       {
         "severity": "error",
-        "code": "deleted",
+        "code": "timeout",
```

Comment:
`timeout` is not a valid code. See http://build.fhir.org/valueset-issue-type.html; maybe just use `processing`

Reply:
fixed here in the table https://github.com/HL7/bulk-data/pull/32/commits/48e1eec7c20d54e9e817ff65bd86aa370690a4c5 (the example i agreed to delete bc we have the dupe of the example in the table)